geoportailv3/static/js/measure/measuredirective.js

```diff
@@ -129,6 +130,8 @@ app.MeasureController = function($scope, $q, $http, ngeoDecorateInteraction,
   this.map_ = this['map'];
 
   var measureProfile = new ngeo.interaction.MeasureLength({
+    startMsg: goog.dom.getElement('measure-profile-start-message'),
```

Comment:
But will the automatic extractor find the strings to translate if they're defined in the javascript?